frontend/packages/pipelines-plugin/src/components/pac/PacPage.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+import { useTranslation } from 'react-i18next';
+import { useLocation, match as Rmatch } from 'react-router-dom';
+import NamespacedPage, {
+  NamespacedPageVariants,
+} from '@console/dev-console/src/components/NamespacedPage';
+import { useAccessReview } from '@console/dynamic-plugin-sdk';
+import { ErrorPage404 } from '@console/internal/components/error';
+import { LoadingBox, AccessDenied, history } from '@console/internal/components/utils';
+import { useFlag } from '@console/shared/src';
+import { FLAG_OPENSHIFT_PIPELINE } from '../../const';
+import { PIPELINE_NAMESPACE } from '../pipelines/const';
+import { usePacData } from './hooks';
+import PacForm from './PacForm';
+import PacOverview from './PacOverview';
+
+type PacPageProps = {
+  match: Rmatch<any>;
```

Comment:
Avoid using `any` type.
```suggestion
  match: Rmatch<{
    ns?: string;
  }>;
```